### ISSUE TITLE
voodoo buggy demoscene demos (minor value, because games are ok)

### ISSUE THREAD

**Issue by WhiteMagicRaven** [joncampbell123/dosbox-x#2715]
artifacts on demo (while on voodoo=opengl) Glide emulation trough opengl

**Expected behavior**
https://www.youtube.com/watch?v=8UZh_s84uRs&ab_channel=MichaelHuth

Windows 7 64 bit
DOSBox-X latest
voodoo_card   = opengl
voodoo_maxmem = true
glide         = true
lfb           = full
splash        = false

(statically linked)
Dimen https://www.pouet.net/prod.php?which=2182 -> minor (3dfx logo upside down, 3dfx logo sometimes wrong texture, wrong surface sorting, completely missed polygons, 2d pictures serious slowdowns)

Other tests (statically linked)
https://www.pouet.net/prod.php?which=16129 (everything ok)

Test of games (statically linked) (all ok, minor slowdowns but nothing affect gameplay, all slowdowns miracle not in 3d scenes)
Battle Arena Toshinden
Descent II
EF 2000
EF 2000: Tactcom
Screamer 2
Starfighter 3000
Tomb Raider
UEFA Champions League
Whiplash
XCar: Experimental Racing
VR Soccer '96: Semifinals
Actua Soccer: Semifinals 3d
Fatal Racing 3d

**Comment by Wengier** [comment]
I think it is important for the purpose of DOSBox-X, as demos are useful to verify the accurateness of the emulation. @joncampbell123 probably wants to take a look at this.

**Comment by grapeli** [comment]
The problem is the second part of this demo, from the moment of changing the brightness of the graphic background. With `core=dynamic_x86` set, it seems to die completely. However, with `core=normal`, it is very slow (about 8-10 minutes for me). After the end of this sequence, the demo continues to play correctly.

**Comment by WhiteMagicRaven** [comment]
https://www.pouet.net/prod.php?which=2844

Trip a dos version makes dosbox x crash when trying to run it

**Comment by grapeli** [comment]
Seems to be working properly from start to finish (`voodoo_card=opengl glide=false`)
I include part, the whole thing will not fit.

https://user-images.githubusercontent.com/452325/126910322-c3dfd694-a1a6-4831-8cdd-f4a0f065a351.mp4

**Comment by WhiteMagicRaven** [comment]
hmm for me it crashes :+1: 
Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	dosbox-x.exe
  Application Version:	0.0.0.0
  Application Timestamp:	60b5f959
  Fault Module Name:	glide2x.dll
  Fault Module Version:	2.60.0.658
  Fault Module Timestamp:	5d7d6fb9
  Exception Code:	c0000005
  Exception Offset:	0000c81f
  OS Version:	6.1.7601.2.1.0.256.1
  Locale ID:	1033
  Additional Information 1:	0a9e
  Additional Information 2:	0a9e372d3b4ad19135b953a78882e789
  Additional Information 3:	0a9e
  Additional Information 4:	0a9e372d3b4ad19135b953a78882e789

Read our privacy statement online:
  http://go.microsoft.com/fwlink/?linkid=104288&clcid=0x0409

If the online privacy statement is not available, please read our privacy statement offline:
  C:\Windows\system32\en-US\erofflps.txt

**Comment by grapeli** [comment]
@Wengier 
This demo dimen behaves weird.
When a scene occurs with lightning dosbox-x as if going into idle mode, the cpu load drops. The window seemed to be losing activity (1:09). If we start to manually change its activity quickly, the demo action runs more smoothly, and after its completion it is completely correct (2:18). Core settings have no effect on this. You can use dynamic_x86.

https://user-images.githubusercontent.com/452325/127734392-6c394afe-8a81-4688-bd64-16509b0acc3e.mp4

**Comment by Wengier** [comment]
@grapeli Yes, I can reproduce the said problem, in both low-level and high-level 3dfx emulation. Not sure if there is some setting (other than core) which can solve this. There is no entry for this particular demo in the Wiki I think.
